cfy_manager/status_reporter/status_reporter.py

```diff
@@ -91,9 +91,15 @@
           help='The status reporter\'s logging level, default level is info'
                'Example: `error`.'
           )
+@argh.arg('--request-timeout',
+          type=int,
+          help='The timeout in seconds that the status reporter will wait for '
+               'an answer from the Cloudify manager.'
+          )
 @argh.arg('-v', '--verbose', help=VERBOSE_HELP_MSG, default=False)
 def configure(managers_ips=None, user_name='', token='', ca_path='',
-              reporting_freq=None, node_id='', log_level='', verbose=False):
+              reporting_freq=None, node_id='', log_level='', verbose=False,
+              request_timeout=1):
```

Comment:
let's init 'request_timeout' with None like 'reporting_freq', so it won't always right this value to the conf